### DISCUSSION TITLE
Loading a video

### DISCUSSION THREAD

**Discussion by jnading** [requarks/wiki#3016] [Help / Questions]
I've tried to load a 5mb video into a document using the video loader in the editor toolbar, but I can't figure out which URL it wants. I know I can't pull from GoogleDrive yet. I saved the video as an asset in the Wiki but the URL to that doesn't work. 

What am I doing wrong? How can I get an MP4 to load and play in a wiki page?

**Comment by NGPixel** [comment]
Video files are not really supported as assets right now. You should instead use an embeddable video like youtube / vimeo.

Proper support is coming in a future release.

**Reply by lordmarqui** [reply]
Are embedded videos working for you? For some reason I cannot get this to work on my instance. I've followed all tutorials to enable iframe use without success. Ideas?

**Reply by jnading** [reply]
We were able to get videos in using the coded page. We save the video as an asset and then use code like this example below (had to reformat so it would show here:

`<video width="640" height="400" controls><source src="https://help.xxxxxx.com/xxxxxx.mp4" `type="video/mp4">`
`</video>`

We can't seem to get this to work on a WYSIWYG-style page though.

Hope this helps

**Reply by ghost** [reply]
I added a module to leverage the `markdown-it-html5-embed` library in #3793. This would allow you to embed audio and video assets in a Wiki.js page using the Markdown image or even link syntax.

**Reply by Nuuki9** [reply]
> I added a module to leverage the `markdown-it-html5-embed` library in #3793. This would allow you to embed audio and video assets in a Wiki.js page using the Markdown image or even link syntax.

It sounds like you've done all the hard work here. Is there any chance you could provide a markdown example for playing an audio file saved as a local asset, as I'm struggling with the right syntax.

**Comment by ohathaway** [comment]
We were able to use a raw video tag with the video files hosted on a public aws bucket. Really, really easy.

Here's a copy/paste of what we added to a Markdown page:
```
<video width="640" height="480" controls preload="auto" poster="https://prs-video-assets.s3.us-west-2.amazonaws.com/img/PRS_Logo_296_76.png">
	<source src="https://prs-video-assets.s3.us-west-2.amazonaws.com/howto/Use+different+web+browsers.webm" type="video/webm">
	<source src="https://prs-video-assets.s3.us-west-2.amazonaws.com/howto/Use+different+web+browsers.mp4" type="video/mp4">
	<source src="https://prs-video-assets.s3.us-west-2.amazonaws.com/howto/Use+different+web+browsers.ogg" type="video/ogg">
	Your browser does not support the video tag.
</video>
```

**Reply by wangjueszu** [reply]
but it just can use to "Ogg\MP4\WebM"
how to use to .flv
